### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by palmfjord**
Makes it possible to trace errors between requests without logging i.e. IPs, adds some timestamps, and avoids console.log which is awesome.

Example; This is what we'd get if we throw the following error in `App.tsx` `throw new Error('expected')`, simulating a render error in react

```
2018-10-03 11:24:45,672 INFO [app] Booting server on 8080 👢
2018-10-03 11:24:45,676 INFO [app] Not using any auth, server is open to the public
2018-10-03 11:24:45,681 INFO [app] Server started 🚀 listening on port 8080

  Error: expected
      at /Users/johanpalmfjord/dev/projects/web-onboarding/build/webpack:/src/App.tsx:17:76
      at App (/Users/johanpalmfjord/dev/projects/web-onboarding/build/webpack:/src/App.tsx:17:62)
      at processChild (/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:2458:14)
      at resolve (/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:2384:5)
      at ReactDOMServerRenderer.render (/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:2706:22)
      at ReactDOMServerRenderer.read (/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:2680:23)
      at renderToString (/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:3071:25)
      at _callee$ (/Users/johanpalmfjord/dev/projects/web-onboarding/build/webpack:/src/server/page.tsx:36:19)
      at tryCatch (/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/regenerator-runtime/runtime.js:62:40)
      at Generator.invoke [as _invoke] (/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/regenerator-runtime/runtime.js:296:22)

2018-10-03 11:24:46,565 ERROR [requestUuid="2b4a3600-c6ee-11e8-aadd-953850855f83":request] Uncaught error in request
Error: expected
@
{anonymous}()@/Users/johanpalmfjord/dev/projects/web-onboarding/build/webpack:/src/App.tsx:17:76
  App()@/Users/johanpalmfjord/dev/projects/web-onboarding/build/webpack:/src/App.tsx:17:62
  processChild()@/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:2458:14
  resolve()@/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:2384:5
  ReactDOMServerRenderer.render()@/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:2706:22
  ReactDOMServerRenderer.read()@/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:2680:23
  renderToString()@/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/react-dom/cjs/react-dom-server.node.development.js:3071:25
  _callee$()@/Users/johanpalmfjord/dev/projects/web-onboarding/build/webpack:/src/server/page.tsx:36:19
  tryCatch()@/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/regenerator-runtime/runtime.js:62:40
  Generator.invoke [as _invoke]()@/Users/johanpalmfjord/dev/projects/web-onboarding/node_modules/regenerator-runtime/runtime.js:296:22
2018-10-03 11:24:46,642 INFO [requestUuid="2b4a3600-c6ee-11e8-aadd-953850855f83":request] http GET /hedvig - 500
```

### CODE REVIEW

File: src/server/middleware/enhancers.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { Middleware } from 'koa'
+import * as uuidV1 from 'uuid/v1'
```
Comment (hedvigoscar) [inline src/server/middleware/enhancers.ts]: Use UUIDv4 instead 🙂

Comment (palmfjord) [reply src/server/middleware/enhancers.ts]: Why? :D v1 has both time and random while v4 has only random?

Comment (hedvigoscar) [reply src/server/middleware/enhancers.ts]: uuidv1 has mac, time and a bit of random, right?

I'd argue that we do not need this added bit of uniqueness traded for 'leaking' this information (then again, this is bikeshedding omega)

Comment (palmfjord) [reply src/server/middleware/enhancers.ts]: hehe, yeh maybe. I think mac adress would be the least of our concerns if our logs leak, but i see your point. expect a push in 30 seconds or so🤘

Comment (hedvigoscar) [reply src/server/middleware/enhancers.ts]: 👏

**Pull request merged**